services/terrain/.gitignore

```diff
@@ -16,10 +16,10 @@
 /.dir-locals.el
 /*.eld
 /target
-/donkey
-/donkey.spec
-/donkey*.tar.gz
-donkey.iml
+/terrain
+/terrain.spec
+/terrain*.tar.gz
+terrain.iml
```

Comment:
I wonder if we should just make this `*.iml`. That would catch all IntelliJ idea files even if the project is named something else. It might not hurt to add `.idea/` to the list also.